automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala

```diff
@@ -73,6 +74,7 @@ trait UserFixtures extends CleanUp with ScaledTimeSpans with Eventually { self:
             None
         }
       })
+      Try(click on find(className("smcx-modal-close")).get)
```

Comment:
does this need a "wait until modal is gone" step?

Reply:
added!